src/binary/ibc-relayer/setup-prometheus.ts

```diff
@@ -0,0 +1,58 @@
+import http from 'http';
+
+import client from 'prom-client';
+
+import { Logger } from '../create-logger';
+
+let initialized = false;
```

Comment:
Hm, do we need some kind of global state?

Reply:
It's just nice to have to make sure we don't initialize the HTTP server more than once: https://github.com/confio/ts-relayer/pull/173/files#diff-95ad111838746b4a0030c8e37ff66c5de9df11b1a2eab5aa3f59895c9f70016cR28-R32

I thought of implementing a singleton here but since it takes arguments like `enabled` I decided to just throw on second call.